components/Header/index.js

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import Button from "../Buttons/Button";
+
+const Header = ({ pageHeader }) => {
+  const { backgroundColour, imgSrc, imgAlt, title, caption } = pageHeader;
+  return (
+    <header>
+      <div className="flex justify-between">
+        <div className="self-center mx-16 w-1/2">
+          <h2 className="py-7 font-bold text-8xl">{title}</h2>
+          <p className=" text-2xl text-gray-500">{caption}</p>
+          <Button caption={"View Categories"} />
```

Comment:
If i wanted to pass an onClick handler to this button component, how would I go about doing that?